firebase.json

```diff
@@ -19,6 +19,10 @@
       "cypress/**"
     ],
     "rewrites": [
+      {
+        "source": "/emailUnsubscribe",
```

Comment:
This rewrite rule means that we can just do `https://cv19assist.com/emailUnsubscribe` or `https://cv19assist-dev.web.app/emailUnsubscribe` in the email templates instead of having the long function url which includes region and could be different between environment